docs/automated-tests/recording/overview.md

```diff
@@ -18,83 +20,83 @@ The below article will guide you trough the recoding process for web and WPF tes
 1. [Attach the Recorder to a Running Application](#attach-the-recorder-to-a-running-application-applicable-for-ie-or-wpf-only)
 1. [What is the Compact Recorder](#what-is-the-compact-recorder)
 
-## __Start a Recording Session__
+## Start a Recording Session
 
-When a web or WPF test is opened and active in the Test Studio project, the toolbar ribbon is switched to the ___Test___ tab. It provides useful options to control the recording and execution process of a test in Test Studio.
+When you open a web or WPF test is and it becomes active in the current Test Studio project, the toolbar ribbon switches to the __Tests__ tab. This tab provides options that control the recording and execution of a test in Test Studio.
 
 ![Test Ribbon Toolbar][2]
 
-Among all available buttons you can see the __Record__ one at the first position. You can start a recording session with a single click on this button.
+You can start a recording session by clicking __Record__.
 
 ![Test Studio][1]
 
 > **Tip**
 > <br>
 > <br>
-> The recording session can be started with a keyboard shortcut **Ctrl+R**, or by using the Project Explorer context menu option __Record__.
+> You can start the recording session with the **Ctrl+R** keyboard shortcut or by using the __Record__ context menu option in the __Project Explorer__.
+
+## Add Steps to a Test
+
+You can use the following approaches to add steps to a test:
 
-## How to Add Steps in the Test
```

Comment:
List items must be grammatically parallel, see https://testdocs.telerik.com/style-guide/lists